IFC4x3/Properties/r/RainIntensity_1HJs_aZFnBKP1bnlrIzkxt/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RainIntensity_1HJs_aZFnBKP1bnlrIzkxt" Name="RainIntensity" UniqueId="514f6fa4-8cfc-4b51-9065-c6fd52f6eef7" PrimaryDataType="IfcLinearVelocityMeasure" />
```

Comment:
Velocity measure? Unexpected. 😄

Reply:
well unit wise technically yes, cause it's the amount of rain that falls (measured in millimetres usually) over time so length/duration. which is technically velocity. but @czapplitec dont forget the semantic content of the measure which is not correct for this case. 

Suggestion: use IfcReal with typical unit details in the description. And i have added it to #336